README.md

```diff
@@ -1,41 +1,29 @@
-# [WIP] skupper [WIP]
+# Skupper
 
-Note: this repository is a work in progress targeted for the consolidation
-and refactoring of the skupper implementation.
+Skupper enables cloud communication by enabling you to create a Virtual Application Network.
 
-Command line tool for setting up and managing skupper installations
+This upper layer network decouples addressing from the underlying network infrastructure, providing a secure and simple alternative to VPNs.
 
-## Usage
+You can use Skupper to create a network from namespaces inn one or more Kubernetes clusters as described in the [Getting Started](https://skupper.io/start/index.html).
+This guide describes a simple network, however there are no restrictions on the topology created which can include redundant paths. 
+You can also create more complex (full mesh) networks, Skupper then supports anycast and multicast communication.
```

Comment:
This last sentence seems to imply that you need a more complex network (perhaps a full mesh) to support anycast and multicast. (Also a full mesh is not what I would mean by a complex network).